README.md

```diff
@@ -22,6 +22,12 @@ To facilitate collaboration on a Serverless Perspective whitepaper/research docu
     docker run --rm -v ~/src/perspective:/source jackwarner/serverless-perspectives example.md --smart --standalone --bibliography sources.bib --csl=/styles/chicago-fullnote-bibliography.csl -o example.pdf
     ```
 
+    If you're running Docker on a Windows machine mind that the path to your repo is in the following format (assumming it's on the C drive, and that the C drive has already been shared with the Docker client):
+
+    ``` 
+    docker run --rm -v /c/path/to/repo/perspective:/source jackwarner/serverless-perspectives example.md --smart --standalone --bibliography sources.bib --csl=/styles/chicago-fullnote-bibliography.csl -o example.pdf
```

Comment:
Another thing I've seen is the suggestion of running the command from the directory in which your markup resides and using `-v %cd%/source` - but I can't test it because docker doesnt work with VMWare :(